src/csharp/Grpc.Tools.CPP.targets

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
```

Comment:
nit: should the Grpc.Tools.* files go under Grpc.Tools/ directory?

Reply:
I expected that, and was going to ask for your suggestion as to where is the best place to put them. Yes, absolutely, I'll move them, thanks!